examples/src/Viewer.tsx

```diff
@@ -795,6 +819,17 @@ class Viewer extends React.Component<InputParams, ViewerState> {
                     Load a smoldyn trajectory
                 </button>
                 <br />
+                <label>
+                    Initial Rabbit Count:
+                    <input
+                        defaultValue="100"
+                        onChange={(event) => {this.smoldynInput = event.target.value}}
+                    />
+                </label>
+                <button onClick={() => this.loadSmoldynSim()}>
+                    Run Smoldyn
+                </button>
+                <br />
```

Comment:
Nit: disable/otherwise indicate when a smoldyn-live sim is not loaded in trajectory select field?